src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -34,48 +34,36 @@ def __init__(self, table: Table):
 
     def box_plots(self) -> Image:
         """
-        Plot a boxplot for every numerical column.
+        Create a box plot for the values in the column. This is only possible for numeric columns.
 
         Returns
         -------
         plot:
-            The plot as an image.
+            The box plot as an image.
 
         Raises
         ------
-        NonNumericColumnError
-            If the table contains only non-numerical columns.
+        TypeError
+            If the column is not numeric.
 
         Examples
         --------
-        >>> from safeds.data.tabular.containers import Table
-        >>> table = Table({"a":[1, 2], "b": [3, 42]})
-        >>> image = table.plot.box_plots()
+        >>> from safeds.data.tabular.containers import Column
+        >>> column = Column("test", [1, 2, 3])
+        >>> boxplot = column.plot.box_plot()
```

Comment:
We're working on a table, not a column here.

Reply:
This was a mistake and is fixed